go/oasis-node/cmd/genesis/migrate.go

```diff
@@ -348,6 +349,18 @@ NodeLoop:
 	// Add a sensible default for the MaxPastRootsStored parameter.
 	newDoc.RootHash.Parameters.MaxPastRootsStored = 1200
 
+	// Update runtime parameters.
+	rts := make([]*registry.Runtime, 0, len(newDoc.Registry.Runtimes)+len(newDoc.Registry.SuspendedRuntimes))
+	rts = append(rts, newDoc.Registry.Runtimes...)
+	rts = append(rts, newDoc.Registry.SuspendedRuntimes...)
+
+	for _, rt := range rts {
+		rt.TxnScheduler.ProposerTimeout = 2 * time.Second
```

Comment:
In order to work around the problem where the 22.2.x runtimes don't allow multiple executions as they don't support same-block verification:
```suggestion
		rt.TxnScheduler.ProposerTimeout = 5 * time.Second
```